certified-connectors/FlowForma V2/README.md

```diff
@@ -0,0 +1,5 @@
+# FlowForma V2
+FlowForma Business Process Management (BPM) tools for Office 365, enable users to build forms & workflow, automating processes without code.
+
```

Comment:
@FlowForma Please refer the template provided for readme.md file and add required sections.